src/components/create-curriculum-component/CreateCurriculumComponent.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Grid, Paper, Checkbox, TextField } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import SearchIcon from '@material-ui/icons/Search';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+
+const icon = <CheckBoxOutlineBlankIcon fontSize="small" />;
+const checkedIcon = <CheckBoxIcon fontSize="small" />;
+
+interface IData {
+    skill: string;
+}
+
+
+
+export function CreateCurriculumComponent() {
+
+    return (
+        <div>
+            <br />
+            <Grid container justify="center">
+                <Paper component="form" >
+
+                    <Autocomplete
+                        popupIcon={<SearchIcon />}
+                        placeholder="test"
+                        multiple
+                        id="checkboxes-tags-demo"
+                        options={skillList}
+                        disableCloseOnSelect
+                        getOptionLabel={(option: IData) => option.skill}
+                        renderOption={(option: IData, { selected }) => (
+                            <React.Fragment>
+                                <Checkbox
+                                    icon={icon}
+                                    checkedIcon={checkedIcon}
+                                    style={{ marginRight: 8 }}
+                                    checked={selected}
+                                />
+                                {option.skill}
+                            </React.Fragment>
```

Comment:
This looks like it should be a skill pill component, that takes in a single skill instead of a material ui checkbox, maybe it should be some combo there in?

Reply:
It will be, but we have not finished that component yet